shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -900,6 +1094,22 @@ - (BOOL)canBecomeFirstResponder {
   return _textInputClient != 0;
 }
 
+- (BOOL)canPerformAction:(SEL)action withSender:(id)sender {
+  // When scribble is available, the FlutterTextInputView will display the native toolbar unless
+  // these text editing actions are disabled.
+  if (isScribbleAvailable() &&
+      (action == @selector(paste:) || action == @selector(cut:) || action == @selector(copy:) ||
+       action == @selector(select:) || action == @selector(selectAll:) ||
+       action == @selector(delete:) || action == @selector(makeTextWritingDirectionLeftToRight:) ||
+       action == @selector(makeTextWritingDirectionRightToLeft:) ||
+       action == @selector(toggleBoldface:) || action == @selector(toggleItalics:) ||
+       action == @selector(toggleUnderline:) || action == NSSelectorFromString(@"_define:") ||
```

Comment:
`_define:` is a private selector?  This may get flagged by iOS app review.

Reply:
That's a good point...we might have to make a whitelist instead of a blacklist to get around that...I'll look into that.

Reply:
Actually, I think we can just return NO for everything – I _think_ the big thing I was worried about was saying no to the accessibility ones, but I just tried it with VoiceOver and it seems to still be able to navigate/read the TextInput.